scripts/opendrive/postinst.sh

```diff
@@ -0,0 +1,23 @@
+echo "Configuring Zimbra-OpenDrive"
+echo "Checking if opendrive zimlet is installed."
+su - zimbra -c "zmmailboxdctl status"
+if [ $? -ne 0 ]; then
+   echo "Mailbox is not running..."
+   echo "Follow the steps below as zimbra user."
+   echo "Remove the opendrive zimlet if it isinstalled."
```

Comment:
typo - missing space inside "isinstalled"

Reply:
Thanks @fatkudu  updated the change.

Reply:
The change was required because the drive zimlet was included but the post install script doesn't have the definition for it to work.